test/app/post_details_test.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:black_tax_and_white_benefits/app/app.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import '../helpers/helpers.dart';
+
+void main() {
+  group('Test Detail view', () {
+    testWidgets('launch url', (tester) async {
```

Comment:
is this name accidental? CHeck out patrol by the way for testing stuff like webviews/lik tapping

Reply:
Linked here: [WA-34](https://anirac-tech.atlassian.net/browse/WA-34?atlOrigin=eyJpIjoiNzRiNjQ1YjdhYzQwNGVjZjljOTM0NDg5Y2RhYzdlMTQiLCJwIjoiaiJ9)